src/client/AM.PA.MonitoringTool/SlackTracker/Views/FirstStartWindow.xaml

```diff
@@ -0,0 +1,32 @@
+﻿<UserControl x:Class="SlackTracker.Views.FirstStartWindow"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:local="clr-namespace:SlackTracker.Views"
+             xmlns:paColors="clr-namespace:Shared;assembly=Shared"
+             mc:Ignorable="d" 
+             d:DesignHeight="300" d:DesignWidth="300">
+    <Grid Background= "White">
+
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        
+        <StackPanel Margin="10" Grid.Row="0">
+            <TextBlock TextWrapping="Wrap">
+                The Slack Tracker is used to collect your activity on different slack channels with your private access token.
```

Comment:
Please explain in a bit more detail what the slack tracker does, what data it collects (since it's really sensitive) and what the benefit of enabling it is. Also, I don't think a regular user understands what a private access token is; not all PersonalAnalytics-users are developers. thanks!